AirCasting/SDSync/Parsing/MiniSDCardMeasurementsParser.swift

```diff
@@ -19,10 +19,11 @@ class MiniSDCardMeasurementsParser: SDMeasurementsParser {
                 if measurementInfo.count == firstLineColumns,
                    let uuidString = getUUID(lineString: lineString) {
                     lastKnownUUID = SessionUUID(rawValue: uuidString)
+                } else {
```

Comment:
This is a fix for a misleading error message, we only want to process the line if it's not a "first line", which contains UUID